sktime/classification/base.py

```diff
@@ -53,6 +63,7 @@ class BaseClassifier(BaseEstimator):
     def __init__(self):
         self.classes_ = []
         self.n_classes_ = 0
+        self.fit_time_ = 0
```

Comment:
May need to add this to all the classifier docs? Maybe not now but a potential issue

Reply:
Indeed, will do so